src/test/java/bio/terra/service/filedata/google/firestore/EncodeFileTest.java

```diff
@@ -383,30 +386,20 @@ public void encodeFileBadRowTest() throws Exception {
     assertThat(
```

Comment:
Can we remove this comment now? Also, should this assertion use the `bucketName` instead of `jade-testdata`? Or will it always be `jade-testdata` in the test?

Reply:
Good catch. I think it `bucketName` is correct.